### PR TITLE
Improve titles of uploaded replays

### PR DESCRIPTION
**Pull request by MacChaeger**
Split off as requested

### CODE REVIEW

**Comment by MacChaeger** [comment]
Actually this is probably pointless since it would be dumb to have replay parsing code in two places in the client so the other PR will end up conflicting with this.

**Comment by Zarel** [comment]
What do you mean?

**Comment by MacChaeger** [comment]
Oh I guess you wanted me to just pull the log directly from the replay server's database instead of having it build all the embedding just to strip it away, which makes a lot of sense now that I understand replays better. In that case, I get why you wanted this split off, but I'm going to need some help with how to access that log.

File: js/client-battle.js [hunk 1]
```diff
@@ -1255,7 +1268,7 @@
 				var index3 = html.indexOf('</script>');
 				html = html.slice(0, index3);
 				html = html.replace(/\\\//g, '/');
-				app.receive('>battle-uploadedreplay\n|init|battle\n|title|Uploaded replay\n' + html);
+				app.receive('>battle-uploadedreplay\n|init|battle\n|title|' + title + '\n' + html);
```
Comment (Zarel) [inline js/client-battle.js]: Should `title` maybe be sanitized?

Comment (Zarel) [reply js/client-battle.js]: I guess it isn't a huge deal.

**Pull request merged**